internal/provider/vaultsecrets/resource_vault_secrets_app_test.go

```diff
@@ -13,6 +14,7 @@ import (
 
 func TestAccVaultSecretsResourceApp(t *testing.T) {
 	testAppName := generateRandomSlug()
+	projectID := os.Getenv("HCP_PROJECT_ID")
```

Comment:
Sorry for the lack on context on my behalf, but will this test only pass if this env var is set?

Reply:
No. The env var exists to set the project the app will be created in. If the app was to be created in the default project, like was happening before, the env var would not be needed.

Reply:
So this is an integration/e2e and not a unit test? It will actually create resources in HCP?

Reply:
Yes, it will

Reply:
should we consider adding two tests here. one with the env, and other with the override in tf files?